### DISCUSSION TITLE
View in Browser size 列使用更好的单位显示

### DISCUSSION THREAD

**Discussion by lilarcor** [rexzhang/asgi-webdav#60] [Ideas]
比如动态地使用KB,MB甚至GB来显示

**Comment by rexzhang** [comment]
这个属于页面上的改动，需要考虑的是如果需要知道详细字节时该怎么办

**Comment by rexzhang** [comment]
这个特性可以完全基于前段实现,欢迎捐献代码(要求使用轻量级前段方案,最好是纯 JS ESx 的)

**Comment by rexzhang** [comment]
可以考虑:当鼠标移动到具体值时以 Tips 的形式显示详细字节信息
